sycl/include/sycl/ext/oneapi/experimental/graph.hpp

```diff
@@ -504,10 +504,26 @@ class command_graph<graph_state::executable>
 namespace detail {
 class __SYCL_EXPORT dynamic_parameter_base {
 public:
+#ifdef __INTEL_PREVIEW_BREAKING_CHANGES
+  dynamic_parameter_base(size_t ParamSize, const void *Data);
+  dynamic_parameter_base();
+#else
   dynamic_parameter_base() = default;
+#endif
+
+#ifndef __INTEL_PREVIEW_BREAKING_CHANGES
+  __SYCL_DEPRECATED("Dynamic_parameter constructors taking a graph object have "
+                    "been deprecated "
+                    "and will be removed in the next ABI breaking window.")
+#endif
```

Comment:
Do we need this, and the same code on line 522-526

Reply:
Ah yes that should be covered by the deprecation warnings on the actual user facing classes, will remove these!